cPP_DSC.adoc

```diff
@@ -948,30 +981,30 @@ FCS_COP.1.1/SigGen:: The TSF shall perform _digital signature generation_ in acc
 
 NIST SP 800-186 (Section 3) [NIST Curves]
 
-{empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018 (Clause 10, 14), FIPS PUB 202#] [SHA]
+{empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018 (Clause 10, 14), FIPS PUB 180-4#] [SHA]
 
 |EdDSA
 |Edwards-Curve Digital Signature Algorithm on {empty}[*selection*: [.underline]#Ed25519 using SHA-512, Ed448 using SHAKE256]#
 |{empty}[*selection*: [.underline]#256, 448]# bits
 |[*selection*: NIST FIPS 186-5 (section 7.6), RFC 8032)
 
-{empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018 (Clause 10, 14), FIPS PUB 202#] [SHA]
+{empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018 (Clause 10, 14), FIPS PUB 180-4, FIPS PUB 202#] [SHA]
 
 |LMS
-|{empty}[*selection*: [.underline]#LMS-OTS, LMS, HSS]#
```

Comment:
LMS-OTS and WOTS+ are one-time signature algorithms and not supposed to be used for general purposes. They are not approved standalone by SP 800-208. They cannot be CAVP tested. Section 9.1 of SP 800-208 has a discussion on the security of one-time signature schemes. In my opinion, they should not be listed in this table.

Reply:
Resolved in #308.